Client/Application/AppDelegate.swift

```diff
@@ -202,7 +204,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.profile = p
         return p
     }
-    
+
+    func setNightModeBrightness(enabled: Bool = false) {
+        let nightModeBrightness: CGFloat = 0.2
+        if (enabled) {
+            systemBrightness = CGFloat(UIScreen.mainScreen().brightness)
+            UIScreen.mainScreen().brightness = nightModeBrightness
+        } else {
+            UIScreen.mainScreen().brightness = systemBrightness
+        }
+    }
+
+    func restoreNightModeBrightness(toForeground: Bool) {
+        let isNightMode = self.profile?.prefs.boolForKey(PrefsKeys.KeyNightModeStatus)
+        if isNightMode == true {
```

Comment:
Night mode should probably be an accessor (a class method in a `NightModeAccessors` class, like `NoImageModeAccessors`. Also see `HomePageAccessors`.

```
class func isNightMode(prefs)
```
```
if isNightMode ?? false {`
```